spec/GameSpec.js

```diff
@@ -0,0 +1,236 @@
+describe('game', function(){
+var game;
+
+  beforeEach(function() {
+    game = new Game();
+  });
+
+  it('has 10 frames on initialization', function(){
+    expect(game._numberOfFrames).toEqual(10);
+  });
+
+  it('knows which frame it is in', function(){
+    expect(game._currentFrameNumber).toEqual(1);
+    expect(game.getTotalScore()).toEqual(0);
+  });
+
+  it('knows which frame it is in after initialization', function(){
+    game.roll(4);
+    game.roll(1);
+    game.getTotalScore()
+    expect(game._currentFrameNumber).toEqual(2);
+    expect(game.getTotalScore()).toEqual(5);
+  });
+
+  it('knows which frame it is in after multiple rolls', function(){
+    game.roll(4);
+    game.roll(1);
+    game.roll(4);
+    game.roll(1);
+    game.roll(4);
+    game.roll(1);
+    game.getTotalScore();
+    expect(game._currentFrameNumber).toEqual(4);
+  });
+
+  it('does not add frame score to total score if the frame has not ended', function(){
+    game.roll(1);
+    game.roll(1);
+    game.roll(1);
+    expect(game.getTotalScore()).toEqual(2);
+  });
+
+  it('does add frame score to total score if the frame has ended', function(){
+    game.roll(1);
+    game.roll(1);
+    game.roll(1);
+    game.roll(0);
+    expect(game.getTotalScore()).not.toEqual(2);
+    expect(game.getTotalScore()).toEqual(3);
+  });
+
+  it('ends a frame if a strike is bowled on the first roll of a frame',function(){
+    game.roll(10);
+    expect(game._currentFrameNumber).toEqual(2);
+  });
+
+  it('does not add the bonus score for a strike if the following frame has not ended', function(){
+    game.roll(10);
+    game.roll(3);
+    game.roll(1);
+    game.roll(1);
+    expect(game.getTotalScore()).not.toEqual(21);
+  });
+
+<<<<<<< HEAD
```

Comment:
It looks like you marked a merge conflict as resolved when it wasn't actually resolved. This type of line should never appear in your committed files.